Source/interfaces/IDisplayInfo.h

```diff
@@ -33,6 +33,34 @@ namespace Exchange {
         virtual uint64_t FreeGpuRam() const = 0;
     };
 
+    struct EXTERNAL IDisplayProperties : virtual public Core::IUnknown {
+        enum { ID = ID_DISPLAY_PROPERTIES };
+
+        virtual ~IDisplayProperties() {}
+
+        struct EXTERNAL INotification : virtual public Core::IUnknown {
+            enum { ID = ID_DISPLAY_PROPERTIES_NOTIFICATION };
+
+            virtual ~INotification() {}
+
+            virtual void ResolutionChanged(const int32_t&, const int32_t&, const string&, const string& ) = 0;
+            virtual void ResolutionPreChange() = 0;
+        };
+
+        virtual uint32_t Register(INotification* sink) = 0;
+        virtual uint32_t Unregister(INotification* sink) = 0;
+
+        virtual string getCurrentResolution() const = 0;
```

Comment:
Coding rules. Use PascalCase for methods.

Reply:
Done